front/app/containers/ProjectsShowPage/shared/header/ProjectActionButtons.tsx

```diff
@@ -51,193 +55,163 @@ interface Props {
-              fontWeight="500"
-            >
-              <FormattedMessage
-                {...getInputTermMessage(inputTerm, {
-                  idea: messages.seeTheIdeas,
-                  option: messages.seeTheOptions,
-                  project: messages.seeTheProjects,
-                  question: messages.seeTheQuestions,
-                  issue: messages.seeTheIssues,
-                  contribution: messages.seeTheContributions,
-                })}
-              />
-            </SeeIdeasButton>
-          )}
-        {process_type === 'continuous' &&
-          participation_method === 'ideation' &&
-          publication_status !== 'archived' && (
-            <IdeaButton
-              id="project-ideabutton"
-              projectId={project.id}
-              participationContextType="project"
-              fontWeight="500"
-            />
-          )}
-        {currentPhase?.attributes.participation_method === 'ideation' &&
-          !hasProjectEnded && (
-            <IdeaButton
-              id="project-ideabutton"
-              projectId={project.id}
-              phaseId={currentPhase.id}
-              participationContextType="phase"
-              fontWeight="500"
-            />
-          )}
-        {surveyPresentOutsideViewport &&
-          ((process_type === 'continuous' &&
-            participation_method === 'survey') ||
-            currentPhase?.attributes.participation_method === 'survey') &&
-          !hasProjectEnded && (
-            <Button
-              buttonStyle="primary"
-              onClick={scrollTo('project-survey')}
-              fontWeight="500"
-            >
-              <FormattedMessage {...messages.takeTheSurvey} />
-            </Button>
-          )}
-        {pollPresentOutsideViewport &&
-          ((process_type === 'continuous' && participation_method === 'poll') ||
-            currentPhase?.attributes.participation_method === 'poll') &&
-          !hasProjectEnded && (
-            <Button
-              buttonStyle="primary"
-              onClick={scrollTo('project-poll')}
-              fontWeight="500"
-            >
-              <FormattedMessage {...messages.takeThePoll} />
-            </Button>
-          )}
-      </Container>
-    );
+  if (isNilOrError(project)) {
+    return null;
   }
 
-  return null;
+  const { process_type, participation_method, publication_status } =
+    project.attributes;
+  const ideas_count =
+    process_type === 'continuous'
+      ? project.attributes.ideas_count
+      : currentPhase?.attributes.ideas_count;
+  const hasProjectEnded = currentPhase
+    ? pastPresentOrFuture([
+        currentPhase.attributes.start_at,
+        currentPhase.attributes.end_at,
+      ]) === 'past'
+    : false;
+  const inputTerm = getInputTerm(
+    project.attributes.process_type,
+    project,
+    phases
+  );
+
+  return (
+    <Container className={className || ''}>
+      {/* {ideasPresentOutsideViewport &&
```

Comment:
I left this commented code as is as I'm not sure why it was removed in the first place

Reply:
I also don't know but let's remove it. This component doesn't seem to exist anymore and the `ideasPresentOutsideViewport` will also not exist with the new changes. No need to leave it there, if it's irrelevant.